config/routes.rb

```diff
@@ -6,13 +6,14 @@ def matches?(request)
 
 Rails.application.routes.draw do
 
-  devise_for :admin, path: 'admin', controllers: {
-    passwords:  'admin/auth/passwords',
-    sessions:   'admin/auth/sessions',
-  }
-
   namespace :admin do
-    get 'site_maps' => 'site_maps#index', as: :root # Devise redirects here after sign in
+    resources :sessions do
+      collection do
+        get 'sign_out' => 'sessions#destroy', as: 'destroy'
+      end
+    end
+
+    get 'site_maps' => 'site_maps#index', as: :root
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.